packages/xdg_directories/pubspec.yaml

```diff
@@ -1,16 +1,16 @@
 name: xdg_directories
 description: A Dart package for reading XDG directory configuration information on Linux.
-version: 0.1.2
+version: 1.0.0-nullsafety.0
```

Comment:
Why have you changed this to 1.0?

Reply:
Saw this in every plugin that got migrated to null safety, that the version updated to next big release.

Reply:
I believe the typical pattern is to go to `0.2.0-nullsafety` instead?